packages/headless-stepper-e2e/src/integration/HeadlessStepper/HeadlessStepper.spec.ts

```diff
@@ -1,7 +1,37 @@
 describe('headless-stepper: HeadlessStepper component', () => {
-  beforeEach(() => cy.visit('/iframe.html?id=headlessstepper--primary'));
-    
-    it('should render the component', () => {
-      cy.get('h1').should('contain', 'Welcome to HeadlessStepper!');
+  beforeEach(() =>
+    cy.visit('/iframe.html?id=headlessstepper-usestepper--use-stepper')
+  );
+
+  it('should render the component', () => {
+    cy.get('h1').should('contain', 'Welcome to HeadlessStepper!');
```

Comment:
This test was failing when I opened Cypress, I had to add this line:

https://github.com/raisiqueira/headless-stepper-monorepo/pull/31/files#diff-2e46bf40d01af0971e8c90c45313e84d23f0d01c933eb5598f8fbab1e3e8263aR28